documentation/modules/exploit/windows/misc/unified_remote_rce.md

```diff
@@ -0,0 +1,313 @@
+## Vulnerable Application
+
+This module utilizes the Unified Remote remote control protocol to type out and
+deploy a payload.  The remote control protocol can be configured to have no passwords,
+a group password, or individual user accounts.  If the web page is accessible, the
+access control is set to no password for exploitation, then reverted.
+If the web page is not accessible, exploitation will be tried blindly.
+
+This module has been successfully tested against version 3.11.0.2483 (50) on Windows 10.
+
+Version 3.11.0.2483 can be downloaded from
+[unifiedremote.com](https://www.unifiedremote.com/static/builds/server/windows-x86/2483/ServerSetup-3.11.0.2483.exe)
+
+## Verification Steps
+
+1. Install the application
+2. Start msfconsole
+3. Do: `use exploit/windows/misc/unified_remote_rce`
+4. Set `rhost` and `lhost` as required.
+5. Do: `run`
+6. You should get a shell.
+
+## Options
+
+
+### WEBSERVER
+
+The port the web server is running on.  Defaults to `9510`
+
+### CLIENTNAME
+
+The name of the client device to use.  This shows up in the Unified Remote logs. If empty
+A random android based name is chosen. Defaults to ``
+
+### SLEEP
+
+The length of time to sleep between each command, this gives the remote program time to process the command on screen.
+Defaults to `1` second.
+
+### PATH
+
+This ONLY applies to the pull method.  Where to temporarily store the payload.  Defaults to `c:\\Windows\\Temp\\`
```

Comment:
What alternatives are there to "the pull method" ?

`pull` is not mentioned or documented anywhere else, with the exception of the module `target` which states "pull".